Code/Engine/Foundation/IO/Implementation/Posix/OSFile_posix.h

```diff
@@ -106,7 +163,10 @@ ezResult ezOSFile::InternalWrite(const void* pBuffer, ezUInt64 uiBytes)
   while (uiBytes > uiBatchBytes)
   {
     if (fwrite(pBuffer, 1, uiBatchBytes, m_FileData.m_pFileHandle) != uiBatchBytes)
+    {
+      ezLog::Error("fwrite 1gb failed for '{}'", m_sFileName);
```

Comment:
1gb??

Reply:
What exactly is the question here? This is a batch write of 1GB size. I wanted to differentiate between this error message and the one a bit further down, so that you can see from the error message if a batch write failed or the final write.